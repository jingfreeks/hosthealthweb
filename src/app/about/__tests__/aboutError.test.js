import { render } from "@testing-library/react";
import AboutError from "../error";

describe("About Error Page", () => {
  it("renders About  Error Page to match snaptshot", () => {
    const { container } = render(<AboutError />);
    expect(container).toMatchSnapshot();
  });
});
