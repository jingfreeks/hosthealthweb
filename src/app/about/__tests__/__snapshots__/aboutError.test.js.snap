// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Error Page renders About  Error Page to match snaptshot 1`] = `
<div>
  <div>
    <h2>
      Something went wrong!
    </h2>
    <button>
      Try again
    </button>
  </div>
</div>
`;
